core/new-gui/src/app/workspace/service/workflow-graph/model/workflow-graph.ts

```diff
@@ -192,6 +192,36 @@ export class WorkflowGraph {
     return Array.from(this.operatorLinkMap.values());
   }
 
+  /**
+   * Returna an array of all input links of an operator in the graph.
+   * @param operatorID
+   */
+  public getInputLinksByOperatorId(operatorID: string): OperatorLink[] {
+    const inputLinks: OperatorLink[] = [];
+    const allLinks: OperatorLink[] = this.getAllLinks();
+    for (let i = 0; i < allLinks.length; i++) {
+      if (isEqual(allLinks[i].target.operatorID, (operatorID))) {
+        inputLinks.push(allLinks[i]);
+      }
+    }
+    return inputLinks;
```

Comment:
There is a spelling error in the comment.

For this method, you can write the function in one line by doing
`return this.getAllLinks().filter(link => link.target.operatorID === operatorID)`

Reply:
If this method is only used for getting the number of input links connected to the current operator, as used in `validation-workflow.service.ts`, then just returned the length of the list instead of the actual list (since you don't use the item in the list anywhere currently).